src/main/java/mekanism/common/item/gear/ItemJetpack.java

```diff
@@ -76,6 +83,9 @@ protected IGasProvider getGasType() {
     public void appendHoverText(@Nonnull ItemStack stack, @Nullable Level world, @Nonnull List<Component> tooltip, @Nonnull TooltipFlag flag) {
         super.appendHoverText(stack, world, tooltip, flag);
         tooltip.add(MekanismLang.MODE.translateColored(EnumColor.GRAY, getMode(stack).getTextComponent()));
+        if (flag.isAdvanced() && Mekanism.hooks.CuriosLoaded && !(stack.is(MekanismTags.Items.CURIOS_BODY) || stack.is(MekanismTags.Items.CURIOS_BACK))) {
+            tooltip.add(MekanismLang.CURIO_NOT_USABLE.translateColored(EnumColor.DARK_GRAY));
+        }
```

Comment:
Having this feels super weird especially based on the wording of the lang entry. I think adding this tooltip should just be removed as if it can go in the curio slots, then curios will add which slots it can go in. And then we also don't have to define these two tags (as currently you are defining them as non-optional even if they are semi optional just due to current data validation.

Reply:
I can keep the tooltip without needing to add those 2 tags as I discovered there's a method which gets the curio tags that an item has. But I can just remove the tooltip if you want.

Reply:
I feel like it should just be removed given the wording of it is a bit odd and also while I don't have a major issue with having optional support for it if people want it I feel like having that message is more of a nag telling people they should heavily consider adding it to a curio slot tag.

Reply:
My though process was that no-one would've known that it works in a Curio slot. Anyways, I'll remove it